cli/jobs/train/r/README.md

```diff
@@ -0,0 +1,40 @@
+---
+page_type: sample
+languages:
+- R
+- azurecli
+products:
+- azure-machine-learning
+description: Learn how to submit R jobs to an AzureMLCompute Cluster using the AzureML 2.0 CLI.
```

Comment:
submit -> create

AzureML -> Azure ML

I would avoid saying "AzureMLCompute Cluster", that seems like an undefined feature/noun in the product